shared/helpers/tests.js

```diff
@@ -18,7 +18,7 @@ const sassPaths = [
 async function compileSassFile(path, options = {}) {
   return compileAsync(path, {
     loadPaths: sassPaths,
-    logger: Logger.silent,
+    silenceDeprecations: ['slash-div'],
```

Comment:
@36degrees Really glad this is out now, I had my eyes on this 🙌

Remember I had to [write my own logger](https://github.com/alphagov/govuk-frontend/blob/adfa0abc33b54262a08d9b2218cd8123a4b0ecf1/shared/tasks/styles.mjs#L137-L143) for Sass builds? Can probably whip that out now too

It's suppressing these two currently:

```js
suppressed: [
  'Using / for division is deprecated and will be removed in Dart Sass 2.0.0.',
  'Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.'
]
```

Reply:
Thanks for the heads up! 🙌🏻

I'll raise an issue as something to do in a follow up PR.